docs/releases/v0.25.0.md

```diff
@@ -0,0 +1,20 @@
+# Changelog
+
+## [v0.25.0](https://github.com/aws-observability/aws-otel-collector/tree/v0.25.0) (2023-01-05)
+
+[Full Changelog](https://github.com/aws-observability/aws-otel-collector/compare/v0.23.1...v0.25.0)
+
+**Closed issues:**
+
+- ECS EC2 sidecar deployment Cloudformation template [\#1591](https://github.com/aws-observability/aws-otel-collector/issues/1591)
```

Comment:
I think we can get rid of this closed issue in favor of pull requests

Reply:
Updated.